src/prometheus/http_test.go

```diff
@@ -4,14 +4,14 @@ import (
 	"net/http"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/stretchr/testify/assert"
```

Comment:
```suggestion
	"github.com/stretchr/testify/require"
	"github.com/stretchr/testify/assert"
```

Reply:
This should have been fixed... somewhere.